astropy/coordinates/tests/test_finite_difference_velocities.py

```diff
@@ -210,6 +210,30 @@ def test_altaz_diffs():
     assert np.all(np.sum(cdiff.d_xyz**2, axis=0) ** 0.5 > constants.c)
 
 
+def test_dt_function():
+    gcrs_coord = GCRS(
+        ra=12 * u.deg,
+        dec=47 * u.deg,
+        distance=100 * u.au,
+        pm_ra_cosdec=0 * u.marcsec / u.yr,
+        pm_dec=0 * u.marcsec / u.yr,
+        radial_velocity=0 * u.km / u.s,
+        obstime=Time("J2020"),
+    )
+    with frame_transform_graph.impose_finite_difference_dt(
+        lambda fromcoord, toframe: 1 * u.s
+    ):
+        icrs_coord = gcrs_coord.transform_to(ICRS())
+    assert_quantity_allclose(
+        icrs_coord.cartesian.xyz, [66.535756, 15.074734, 73.518509] * u.au
+    )
+    assert_quantity_allclose(
+        icrs_coord.cartesian.differentials["s"].d_xyz,
+        [-29.93183, -4.715867, -2.103387] * u.km / u.s,
```

Comment:
Are these just from the default? If so, may as well note (i.e., that this is an internal test only).

Reply:
The test now has a short docstring.